data/Canham_2023/metadata.yml

```diff
@@ -170,23 +170,24 @@ contexts:
   category: plot
   var_in: Inundation (days)
   values:
-  - value: 2
-  - value: 3
-  - value: 4
-  - value: 5
-  - value: 6
-  - value: 10
-  - value: 11
-  - value: 12
-  - value: 15
-  - value: 18
-  - value: 19
-  - value: 31
-  - value: 34
-  - value: 36
-  - value: 37
-  - value: 38
-  - value: '00'
+  - value: 38.0
+  - value: 10.0
+  - value: 6.0
+  - value: 5.0
+  - value: 3.0
+  - find: 0.0
```

Comment:
How come this needs to be converted to character?

Reply:
Not sure - it was a change you made on a branch that I'm deleting... I simply copied across the change

Reply:
Oh well I guess may as well keep it for now. I feel like with the fixes to traits.build it shouldn't prove to be an issue in the future as I'm sure I convert everything to character now